core/src/main/java/org/commonjava/maven/ext/core/ManipulationManager.java

```diff
@@ -191,6 +192,10 @@ public void init( final ManipulationSession session )
     public void scanAndApply( final ManipulationSession session )
                     throws ManipulationException
     {
+        final PreparseGroovyManipulator preparseGroovyManipulator = new PreparseGroovyManipulator( null, null, pomIO );
+        preparseGroovyManipulator.init( session );
+        preparseGroovyManipulator.applyChanges( Collections.emptyList() );
+
```

Comment:
I placed the call inside `scanAndApply` so that if called in code (and not CLI) the preparse would still be run.

Reply:
While initially I did think the CLI was a _more_ appropriate place, thinking on it further, if its in `scanAndApply` then it will work whether PME is run as a CLI **or** as an extension.